front/src/modules/companies/constants/companyViewFields.tsx

```diff
@@ -111,4 +116,41 @@ export const companyViewFields: ViewFieldDefinition<ViewFieldMetadata>[] = [
     },
     isVisible: true,
   } satisfies ViewFieldDefinition<ViewFieldTextMetadata>,
+  {
+    id: 'idealCustomerProfile',
+    columnLabel: 'ICP',
+    columnIcon: <IconTarget />,
+    columnSize: 150,
+    columnOrder: 8,
+    metadata: {
+      type: 'boolean',
+      fieldName: 'idealCustomerProfile',
+    },
+    isVisible: false,
+  } satisfies ViewFieldDefinition<ViewFieldBooleanMetadata>,
+  {
+    id: 'annualRecurringRevenue',
+    columnLabel: 'ARR',
+    columnIcon: <IconMoneybag />,
+    columnSize: 150,
+    columnOrder: 8,
+    metadata: {
+      type: 'moneyAmount',
+      fieldName: 'annualRecurringRevenue',
+    },
+    isVisible: false,
```

Comment:
My understanding of [the issue](https://github.com/twentyhq/twenty/issues/861) is that this field should be visible by default.
Looking good otherwise 👍